README.md

```diff
@@ -49,6 +49,9 @@ jobs:
       
       - name: Delete zips
         run: rm -f *.zip
+        
```

Comment:
I suppose first you would have to delete the existing `docs` directory and then rename the `html` directory